charts/metering-ansible-operator/templates/crds/hive.crd.yaml

```diff
@@ -21,3 +21,225 @@ spec:
+                Generally, this field should be set to "default", or the value of the "databaseName" in a Hive StorageLocation.
+                More info: https://github.com/operator-framework/operator-metering/blob/master/Documentation/storagelocations.md
+              minLength: 1
+            tableName:
+              type: string
+              description: |
+                TableName is the desired name of the table to be created in Hive.
+              minLength: 1
+            columns:
+              type: array
+              description: |
+                A list of columns that match the schema of the HiveTable.
+                For each list item, you must specify a `name` field, which is the name of an individual column for the Hive table,
+                and a `type` field, which corresponds to a valid type in Hive.
+                Note: the only complex types supported are map's of primitive types.
+                More info: https://cwiki.apache.org/confluence/display/Hive/LanguageManual+Types
+              items:
+                type: object
+                required:
+                - name
+                - type
+                properties:
+                  name:
+                    type: string
+                    description: |
+                      The name of the column.
+                    minLength: 1
+                  type:
+                    type: string
+                    description: |
+                      The column data type.
+                    minLength: 1
+            partitionedBy:
+              type: array
+              description: |
+                A list of columns that are used as partition columns.
+                Columns in "partitionedBy" and "columns" must not overlap.
+                For each list item, you must specify both a `name` and `type` field.
+                Note: this is an optional field.
+              items:
+                type: object
+                required:
+                - name
+                - type
+                properties:
+                  name:
+                    type: string
+                    description: |
+                      The name of the column.
+                    minLength: 1
+                  type:
+                    type: string
+                    description: |
+                      The column data type.
+                    minLength: 1
+            clusteredBy:
+              type: array
+              description: |
+                A list of columns from "columns" to use for bucketed tables.
+                This field must be set if "numBuckets" is specified.
+                Note: this is an optional field.
+                More info: https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL+BucketedTables
+              items:
+                type: object
+                required:
+                - name
+                - type
+                properties:
```

Comment:
@chancez there wasn't much info on this field in the documentation, besides that you need to list the columns to use for bucketed fields. Is this just an array of column names, or does it emulate the `columns` property (where it's an array and each array element is an object.)

Reply:
It looks like it's the fomer: `ClusteredBy   []string     `json:"clusteredBy,omitempty"``

Reply:
https://github.com/operator-framework/operator-metering/blob/master/pkg/apis/metering/v1/hive_table.go#L39